src/classic/types/ReactPropTypes.js

```diff
@@ -86,19 +86,20 @@ var ReactPropTypes = {
 };
 
 function createChainableTypeChecker(validate) {
-  function checkType(isRequired, props, propName, componentName, location) {
+  function checkType(isRequired, props, propName, componentName, location, propFullName) {
```

Comment:
I'm also not wild about parameter order now, it might be time to make these into a single object arg (though we'll break custom validators). Though this is safest for now.